lib/errors/spec.js

```diff
@@ -85,8 +85,14 @@ module.exports = [{
     name: 'InvalidSatoshis',
     message: 'Output satoshis are invalid',
   }, {
-    name: 'FeeError',
-    message: 'Fees are not correctly set {0}',
+    name: 'SmallFeeError',
+    message: 'Fee is too small: {0}',
+  }, {
+    name: 'LargeFeeError',
+    message: 'Fee is too large: {0}',
+  }, {
+    name: 'DifferentFeeError',
+    message: 'Unspent value is different from specified fee: {0}',
   }, {
```

Comment:
Perhaps we could make these additional errors part of a general `FeeError`:

```js
if (err instanceof errors.FeeError) {
  //...
}
```
And if interested in a more specific exception:
```js
if (err instanceof errors.FeeError.TooSmall) {
  //...
}
```